crashDataAnalysisTools/merge_elev.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+import pandas as pd
+import sqlite3 as dbi
+
+def get_seg_elev(road_tb, elev_tb):
+    '''
+    This is the function that integrate the grade information stored in the
+    elevation table into the road segment table.
+
+    Two .csv files were saved in the current working directory, with Washington
+    State road segment data and elevation information stored separately.
+    The function creates a database which contains these two tables together
+    and executes a sql query to merge these two tables.
+
+    The input of the function include two string variables denote the file
+    names (including file paths if needed) of the two data tables. The output
+    of the function will be the merged table in the form of a pandas dataframe.
+    '''
+
+    # read the data tables from .csv files
+    roads = pd.read_csv(road_tb)

```

Comment:
Do you want the user to specify the full file path? Or just the relative file path based on the working directory (if this, need to use os.getcwd())?